Source/Fuse.Reactive.JavaScript/FuseJS/Observable.js

```diff
@@ -1679,4 +1679,14 @@ Observable._getDataObserver = function(node, key)
 	return res;
 }
 
+Observable.prototype.pull = function(onResult){
+	var self = this;
+	function resolver(obs)
+	{
+		onResult(obs);
+		self.removeSubscriber(this);
+	}
+	self.addSubscriber(resolver);
```

Comment:
This needs to be attached to the `module` somehow to ensure it is cleanedup even if resolve is not called.